libraries/dojo/package.json

```diff
@@ -4,10 +4,11 @@
     "build": "npm run test; DIST=../../docs/libraries/dojo && mkdir -p $DIST && cp -r ./meta $DIST/meta && cp -r ./results $DIST/results"
```

Comment:
Would you mind updating line 3 to change the text from `@dojo/core` to `@dojo/framework`? This is used to display the version number next to the library name on the site.